docs/_interface/mining.md

```diff
@@ -53,7 +53,7 @@ Now we're ready to start mining. In a new terminal session, run ethminer and con
 
 `ethminer` communicates with geth on port 8545 (the default RPC port in geth). You can
 change this by giving the [`--rpcport` option](../rpc/server) to `geth`. Ethminer will find
-get on any port. You also need to set the port on `ethminer` with `-P
+geth on any port. You also need to set the port on `ethminer` with `-P
```

Comment:
Thanks, but.. While you're fixing this, there are some other errors that should be fixed aswell. The `etherbase`, I think is called `--miner.etherbase`, and `--rpcport` is `--http --http.port`. I suspect this page is pretty bitrotted and needs some loving